src/Aliencube.AzureFunctions.FunctionAppV1/SampleHttpTrigger.cs

```diff
@@ -70,5 +70,7 @@ public static class SampleHttpTrigger
 
             return result;
         }
+
+
     }
```

Comment:
Probably those empty lines are left here by an accident. In such case they should be removed.